docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -1048,8 +1086,140 @@ def training_step(self, batch, batch_index):
         return loss
 
     def configure_optimizers(self):
+        # Training diverges without LARS
+        optim = LARS(
+            self.parameters(),
+            lr=0.3 * lr_factor,
+            weight_decay=1e-4,
+            momentum=0.9,
+        )
+        scheduler = torch.optim.lr_scheduler.LambdaLR(optim, self.scale_lr)
+        return [optim], [scheduler]
+
+    def scale_lr(self, epoch):
+        if epoch < self.warmup_epochs:
+            return epoch / self.warmup_epochs
+        else:
+            return 0.5 * (
+                1.0
+                + math.cos(
+                    math.pi
+                    * (epoch - self.warmup_epochs)
+                    / (max_epochs - self.warmup_epochs)
+                )
+            )
```

Comment:
Why did you have to change the `VICRegModel` as well? And I saw there were also some changes to `SimMIM`.

Reply:
It's only formatting and adding a comment